model/src/pyrenew/metaclass.py

```diff
@@ -95,14 +95,83 @@ def _assert_sample_and_rtype(
 class RandomVariable(metaclass=ABCMeta):
     """
     Abstract base class for latent and observed random variables.
+
+    Notes
+    -----
+    `RandomVariable`s in `pyrenew` can be time-aware, meaning that they can
```

Comment:
this line does not render properly 

<img width="749" alt="Screenshot 2024-07-03 at 4 29 52 PM" src="https://github.com/CDCgov/multisignal-epi-inference/assets/8032117/108d447f-1a32-4b46-a89c-da3c02a9e59e">

Reply:
Renders properly as of 4bfa988